readme.md

```diff
@@ -127,6 +127,26 @@ Custom promise module to use instead of the native one.
 - Pify has useful options like the ability to handle multiple arguments (`multiArgs`).
 - Pify does not have [magic behavior](https://nodejs.org/api/util.html#util_custom_promisified_functions) for certain Node.js methods and instead focuses on predictability.
 
+#### How can I promisify a single class method?
+
+Class methods are not bound, so when they're not called on the class itself, they don't have any context. You can either promisify the whole class or use `.bind()`.
+
+```js
+const pify = require('pify');
+const SomeClass = require('./some-class');
+
+const someClass = new SomeClass();
+
+// `someFunction` can't access its class context.
+const someFunction = pify(someClass.someFunction);
+
+// `someFunction` is extracted from the entire promisified class, allowing it to keep its context.
+const someFunction = pify(someClass).someFunction;
+
+// `someFunction` is bound to its class.
+const someFunction = pify(someClass.someFunction).bind(someClass)
```

Comment:
This is not what I was suggesting. Also missing semicolon.

Reply:
What were you suggesting?

Reply:
You could have checked this yourself, but it's here: https://github.com/sindresorhus/pify/pull/80#discussion_r429123145 The `.bind` should be on the original method, not the promisified one.

Reply:
Fixed!